docs/reference/server-configuration.md

```diff
@@ -10,13 +10,22 @@ keywords: ksqldb, configure, server, setup, install
 
 These configuration parameters control the general behavior of ksqlDB server.
 Many parameters can only be set once for the entire server, and must be
-specified using the `ksql-server.properties` file.
+specified using the `ksql-server.properties` file (for on-prem / standalone).
+In this case, configurations are applied when the cluster starts.
+
+A subset of these configuration parameters can be applied on a running cluster,
+either for individual queries (using the SET command or the Confluent Cloud
+Console) or for the entire cluster (using the ALTER SYSTEM command or the
```

Comment:
maybe note here that `ALTER SYSTEM` is only available on Confluent Cloud ie not yet applicable on Confluent platform/on-prem installations?

Reply:
Also, and maybe this is just me being ignorant, but what do you mean by "Confluent Cloud Console" -- I'm guessing that's the UI, is that our official term for it? I think I'm just a bit thrown off by the "console" part of it, that always makes me think 'terminal' or 'command line' interface. But if this is established elsewhere in the docs and would be clear to our users in your opinion then just ignore me 🙂

Reply:
@ableegoldman Yes, "Confluent Cloud Console" is the official name for the Cloud UI, and I agree, it's confusing. We should go with the industry standard and call it "Confluent Cloud **Portal**".